### DISCUSSION TITLE
Merge playlist from different streaming services

### DISCUSSION THREAD

**Discussion by lor74** [music-assistant/support#1665] [Q&A] [answered]
It's possible to merge/create a playlist from more streaming services?
For examples I like White Buffalo but on qobuz not all the songs are available compared to spotify. I would create a playlist with all my favourite tracks using qobuz and add the missing from spotify.
Thanks for support.

**Comment by OzGav** [comment, marked as answer]
You can’t merge playlists but you can create a playlist that has tracks from multiple providers.

**Comment by lor74** [comment]
Hi, thanks... indeed it's super easy! 
I have followed this hint #661, create an empty playlist under /media (but now I will migrate on a NAS folder)... and I could add the track in the playlist. What I have missed was how to create a playlist.

Great! 👍

**Reply by OzGav** [reply]
Since that post you can now create a new playlist from with MA

![IMG_1165](https://github.com/music-assistant/support/assets/19848947/bed06f37-b60b-400c-aae2-6c0dcc042675)

**Reply by lor74** [reply]
Thanks... it works.
An .m3u file is created on NAS ... without filename... but I have to click two times on the button reported aobve.

```
drwxrwsr-x+ 2 lorenzo allusers 4.0K Dec 30 08:42 .
drwxrwxrwx+ 4 root    root     4.0K Dec 30 08:37 ..
-rwxrwxr--+ 1 lorenzo allusers   54 Dec 30 08:43 .m3u
-rw-rwxr--+ 1 lorenzo allusers  282 Dec 29 15:54 ma_white_buffalo.m3u
lorenzo@TNAS-NERONI:/mnt/md0/HA/MA$ pwd
/mnt/md0/HA/MA
```
This playlist is overwitten every time "Create new playlist on" is selected (two times)
Could be possible to rename the playlist from GUI?

**Reply by OzGav** [reply]
You can it’s just not completely obvious. You can type in a name when you create the playlist

**Reply by lor74** [reply]
Thanks... yes clicking on the name not on the button... it works fine! 👍
